### PR TITLE
cmd/scollector: Chef collector

### PR DESCRIPTION
**Pull request by elementai**
Simple collector for Chef, reads and parses last line from `/var/chef/reports/run_status_json.log`.

### CODE REVIEW

File: cmd/scollector/collectors/chef_linux.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	chefRunLog = "/var/chef/reports/run_status_json.log"
+)
+
+func chefEnable() bool {
+	_, err := os.Stat(chefRunLog)
+	return err == nil
+}
+
+func c_chef_linux() (opentsdb.MultiDataPoint, error) {
+	md, err := chef_linux(chefRunLog)
```
Comment (elementai) [inline cmd/scollector/collectors/chef_linux.go]: Additional function is for tests convenience. Will include tests later, need to discuss a bit.

File: cmd/scollector/collectors/chef_linux.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"bufio"
```
Comment (mjibson) [inline cmd/scollector/collectors/chef_linux.go]: This should be grouped with other stdlib imports.

File: cmd/scollector/collectors/chef_linux.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type CRSummary struct {
```
Comment (mjibson) [inline cmd/scollector/collectors/chef_linux.go]: Make this unexported.

**Comment by mjibson** [comment]
@kylebrandt ok adding this? LGTM.

**Pull request merged**